### DISCUSSION TITLE
Face recognize stopped

### DISCUSSION THREAD

**Discussion by freedirect** [nextcloud/recognize#973] [Q&A]
I have installed recognize on my nextcloud server and it seems to work correctly, it detects the faces. The problem is that it seems to have finished the process but it has not inspected all the photos available on the nextcloud server.

Would anyone know how to make it continue inspecting all the photos?

Thank you very much.

**Comment by marcelklehr** [comment]
> it has not inspected all the photos available on the nextcloud server

How do you come to this conclusion?

**Reply by freedirect** [reply]
Sorry for not giving more information.

The thing is that when I go to photos -> people, I see that the app has recognised many of the people in the photos. But if I go to any of the people who have the most matches, for example my face, I see that the last photo detected is from a few years ago. As I have the photos sorted by year and month folders, I understand that the process has stopped at some point.

I don't know if there is a way to find out the photos that the app has been scanning in a log.

Thank you very much.

**Reply by marcelklehr** [reply]
Can you check the admin settings for recognize, perhaps make a screenshot of the stats there? It should give an overview of the status of recognize's operations.

**Reply by freedirect** [reply]
![IMG_20231128_210537](https://github.com/nextcloud/recognize/assets/69584302/7b892bc3-80c0-4155-8fc1-c878709f59d8)

I don't see anything strange, I don't know...
